app/assets/javascripts/pages/agent-edit-page.js.coffee

```diff
@@ -126,17 +129,49 @@ class @AgentEditPage
       else
         @hideEventCreation()
 
+  buildAce: ->
+    $(".ace-editor").each ->
+      unless $(this).data('initialized')
+        $(this).data('initialized', true)
+        $source = $($(this).data('source')).hide()
```

Comment:
I am wondering, why `$source` vs `source`, both would be the same or am I missing something?